src/app/cards/munchkin/cards/treasures/disposable-item/model/disposable-item-card.ts

```diff
@@ -0,0 +1,15 @@
+
+import { DisposableItemType } from './disposable-item-type';
+import { ImageBottomLayout } from 'src/app/cards/infrastructure/cards';
+import { TreasureCard } from '../../treasure-card';
+
+/**
+ * Represents a disposable item card. Extends [[DoorCard]].
```

Comment:
I'm feeling a bit confused: was there supposed to be `Extends [[TreasureCard]]` ?